packages/web/app/src/pages/target-laboratory.tsx

```diff
@@ -328,8 +328,26 @@ function LaboratoryPageContent(props: {
         });
         try {
           const result = await preflightScript.execute();
-          if (result && headers) {
-            headers = substituteVariablesInHeader(headers, result);
+          // todo: Why check `result` truthiness here if it can never by falsy?
+          if (result) {
```

Comment:
Can we remove the `if` check here?

Reply:
If it throws when there is no result, then yes

Reply:
<img width="921" alt="image" src="https://github.com/user-attachments/assets/cbb9ed00-2a0b-4107-b57b-e52427a34865" />

`safeParseJSON` returns `null`, thus the check

Reply:
In my PR a fallback value is now returned. I seemed to have forgotten that the original code did this and confused my self. I will remove the `if` in this PR.